src/Mod/Material/StandardMaterial/Tools/make_FCMats.sh

```diff
@@ -26,7 +26,7 @@ function get_xy() {
 }
 
 # Determine number of columns and rows in the Materials.csv file
-NUMBER_OF_COLUMNS=$(cat $MATERIALS_FILE.csv | awk --field-separator="|" "NR==1 { print NF }")
+NUMBER_OF_COLUMNS=$(cat $MATERIALS_FILE.csv | awk -F "|" "NR==1 { print NF }")
```

Comment:
Hi @dubstar-04 excuse my ignorance here but isn't `-F` and `--field-separator` synonymous and therefore makes this fix redundant?

Per the `awk` documentation it is. 
```
Usage: awk [POSIX or GNU style options] [--] 'program' file ...
POSIX options:          GNU long options: (standard)
        -F fs                   --field-separator=fs
```

And if that is the case then the proposed fix is a workaround for this `awk` anomally?

Reply:
I'm not sure why --field-separator fails and -F works.

Reply:
@dubstar-04 In order to retain your commit, I would add a comment explaining the decision to use the shorthand flag over the longform. This signals to future devs not to change this back.

Reply:
@luzpaz Yes, `-F` is POSIX and supported by all implementations.  `--field-separator` is only supported in GNU awk (typically Linux) so often the long form doesn't work on e.g. macOS which ships the original awk.

Reply:
I agree with @noahwilliamsson. I'd get rid of the `cat` portion and simplify it to a pure `awk` command.

Reply:
Removed the 'cat' portion of the command as suggested. I didn't see the need to comment on the use of the -F rather than --field-seperator as -F is used elsewhere in the script.